nerfstudio/model_components/renderers.py

```diff
@@ -332,3 +332,32 @@ def forward(
         """Calculate normals along the ray."""
         n = torch.sum(weights * normals, dim=-2)
         return n
+
+
+class LambertianShadingRenderer(nn.Module):
+    """Calculate Lambertian shading."""
+
+    @classmethod
+    def forward(
+        cls,
+        rgb: TensorType["bs":..., "num_samples", 3],
+        normals: TensorType["bs":..., "num_samples", 3],
+        light_direction: TensorType["bs", 3],
+        ratio: float = 1.0,
```

Comment:
Seems odd for the default to be non-lambertian. Does something like .5 make more sense?

Reply:
Actually thinking a bit more, I think `ratio` could use a different name like `shading_weight` where when `shading_weight = 1` the render gives you something that is pure lambertian (0 gives no shading). The default can then be 1.0.

Reply:
yeah that name is more clear